mobile-app/app/components/OceanInterface/TransactionError.tsx

```diff
@@ -83,6 +83,26 @@ function errorMessageMapping (err: string): ErrorMapping {
       code: ErrorCodes.InsufficientDFIInVault,
       message: 'Insufficient DFI collateral (≥50%)'
     }
+  } else if (err.includes('Lack of liquidity')) {
+    return {
+      code: ErrorCodes.InsufficientDFIInVault,
```

Comment:
Same error codes for all the newly added one?

Reply:
Oh yeah good point let me change it

Reply:
Updated @kyleleow